os/include/tinyara/pm/pm.h

```diff
@@ -616,12 +616,13 @@ int pm_changestate(int domain, enum pm_state_e newstate);
 
 enum pm_state_e pm_querystate(int domain);
 
+#ifdef CONFIG_PM_DVFS
 /****************************************************************************
  * Name: pm_dvfs
  *
  * Description:
- *   This internal function is called to reduce the frequency of operation
- *   of the secondary AP core of AI_Dual chipset.
+ *   This internal function is called to reduce the clock frequency
+ *   of the AP core (CA32) of AI_Dual chipset.
```

Comment:
1. AI Dual is our internal name. It's not proper to use in public github.
2. I left the comment as it's not proper to add chip-specific API because pm is the common module. With the same reason, `AP core (CA32) of AI_Dual chipset` can be changed as general description.

Reply:
I agree, this function is intended to be used in common for all chipsets later.
So lets not have the description specific to one chipset.

Reply:
Modified to common description, please check~